### DISCUSSION TITLE
How to export web pages that require login to display content?

### DISCUSSION THREAD

**Discussion by RealForMe** [danburzo/percollate#176] [Q&A]

**Comment by danburzo** [comment]
You can fetch the web pages with an external tool and pipe the content into `percollate`, using the `-` operand to signify reading from `stdin` and making sure you pass the original URL with the `--url` option. An example from the README:

```bash
curl https://example.com | percollate pdf - --url=https://example.com
```

You can _Copy as cURL_ the URL containing all the proper parameters using the Network tab in the Developer Tools of your browser.
